apache/zeppelin
# [ZEPPELIN-1273] Use Math.abs to determine if custom formatter should be called for y axis values

### What is this PR for?
Use Math.abs to determine if custom formatter should be called for y axis values.


### What type of PR is it?
Bug Fix

### Todos

### What is the Jira issue?
[ZEPPELIN-1273] https://issues.apache.org/jira/browse/ZEPPELIN-1273

### How should this be tested?
Create simple table with two columns - date in one and large negative numbers like -10000000 in other. Display line chart and see that the y axis values are formatted like -10M.

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? 
No
* Is there breaking changes for older versions?
No
* Does this needs documentation?
No
